daemon/exec/exec.go

```diff
@@ -86,6 +89,30 @@ func (c *Config) SetExitCode(code int) {
 	c.ExitCode = &code
 }
 
+// InitReadyWaitGroup set wait group to 1
+func (c *Config) InitReadyWaitGroup() {
+	c.started = make(chan struct{})
```

Comment:
You can initialize the channel in `exec.NewConfig()`:

```go
func NewConfig() *Config {
	return &Config{
		ID:           stringid.GenerateNonCryptoID(),
		StreamConfig: stream.NewConfig(),
                started:      make(chan struct{}),
	}
}
```

Reply:
copy that~